### PR TITLE
Radar accuracy improvements

### PR DESCRIPTION
**Pull request by Piccirello**

### CODE REVIEW

File: src/dictionaries/custom.json [hunk 1]
```diff
@@ -140,6 +144,7 @@
   "eval",
```
Comment (bvallelunga) [inline src/dictionaries/custom.json]: emacs?

File: src/filters/common_patterns.js [hunk 2]
```diff
@@ -5,13 +5,17 @@ class CommonPatterns extends Filter {
     super('Common programming patterns');
 
     // find chained variables (e.g. foo.bar.method)
-    this.chainedVariables = /^([a-zA-Z0-9]+\.){2,}[a-zA-Z0-9]+$/;
+    this.chainedVariables = /^(?:[a-zA-Z0-9]+\.){2,}[a-zA-Z0-9]+$/;
```
Comment (bvallelunga) [inline src/filters/common_patterns.js]: Can we add Ruby support `MixIn::Test::Fun`

File: src/filters/url.js [hunk 3]
```diff
@@ -10,10 +10,28 @@ const markdownLinkRegex = /\[[\w#-_:`\\/\.]+\]\([\w#-_:`\\/\.]+\)/;
...
     super('URL');
+
+    this.commonTLDs = Array.from(new Set([
+      'com',
+      'org',
+      'net',
+      'gov',
+      'edu',
+      'co.uk',
+      'ru',
```
Comment (bvallelunga) [inline src/filters/url.js]: `io`?

File: src/secrets/api_keys.js [hunk 4]
```diff
@@ -39,7 +39,7 @@ class APIKeys extends Secret {
     this.maxTermLength = 256;
     this.maxLineLength = 512;
 
-    this.excludedTerms = ['regexp', 'shasum', 'http://', 'https://', 'file://', 'data:', 'gitHead', 'function', 'example'];
+    this.excludedTerms = ['regexp', 'shasum', 'http://', 'https://', 'file://', 'data:', 'gitHead', 'function', 'example', 'return'];
```
Comment (bvallelunga) [inline src/secrets/api_keys.js]: Add comment that it is for excluding words with no space in it.

File: src/filters/url.js (hunk 3 above)
Comment (Piccirello) [reply src/filters/url.js]: Added io and some other tlds

**Pull request merged**